raft/raftpb/raft.proto

```diff
@@ -118,6 +118,7 @@ enum ConfChangeType {
 	ConfChangeRemoveNode     = 1;
 	ConfChangeUpdateNode     = 2;
 	ConfChangeAddLearnerNode = 3;
+	ConfChangeDowngrade			 = 4;
```

Comment:
I don't think Downgrade is config change from Raft's perspective, because it is not changing Raft's cluster configuration. Can we add a new type in https://github.com/etcd-io/etcd/blob/master/etcdserver/etcdserverpb/raft_internal.proto

Reply:
I am thinking something generic, so later if we have similar needs to add more cluster info, we can reuse the same type.